mainhandler/imageregistryhandler.go

```diff
@@ -414,23 +427,42 @@ func (registryScan *registryScan) filterScanLimit(limit int) {
 	registryScan.mapImageToTags = filteredImages
 }
 
+func (registryScan *registryScan) enumerateRepoes() ([]string, error) {
+	// token sometimes includes a lot of dots, so we need to remove them
+	i := strings.Index(registryScan.registryInfo.AuthMethod.Password, ".....")
```

Comment:
use [TrimLeft](https://pkg.go.dev/strings#TrimLeft)

Reply:
it won't work, token has single dots before